kali_cloud_init.tf

```diff
@@ -15,7 +15,7 @@ data "template_cloudinit_config" "kali_cloud_init_tasks" {
       "${path.module}/cloud-init/efs-mount.tpl.yml", {
         # Just mount the EFS mount target in the first private subnet
         efs_id      = aws_efs_mount_target.target[var.private_subnet_cidr_blocks[0]].file_system_id
-        mount_point = "/data"
+        mount_point = "/share"
```

Comment:
I see that the appropriate [corresponding change](https://github.com/cisagov/ansible-role-xfce-cool/pull/2) is coming to `ansible-role-xfce-cool`. 👍